examples/gke/xfs-formatter-daemonset.yaml

```diff
@@ -0,0 +1,92 @@
+# Daemonset that converts ephemeral-storage to xfs
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: xfs-formatter
+spec:
+  selector:
+    matchLabels:
+      app: xfs-formatter
+  template:
+    metadata:
+      labels:
+        app: xfs-formatter
+    spec:
+      hostPID: true
+      nodeSelector:
+        scylla.scylladb.com/gke-ephemeral-storage-local-ssd: "true"
+      tolerations:
+      - key: role
+        operator: Equal
+        value: scylla-clusters
+        effect: NoSchedule
+      containers:
+      - name: xfs-formatter
+        image: docker.io/scylladb/scylla-operator:latest
+        securityContext:
+          privileged: true
+        command:
+        - "/bin/bash"
+        - "-c"
+        args:
+        - |
+          set -euExo pipefail
+          shopt -s inherit_errexit
+
+          cp -r /usr/local/lib/scylla-operator/gke/xfs-formatter /host/var/lib/
+
+          if [[ ! -d /host/usr/local/lib/systemd ]]; then
+            mkdir /host/usr/local/lib/systemd
+          fi
+
+          if [[ ! -d /host/usr/local/lib/systemd/system ]]; then
+            mkdir /host/usr/local/lib/systemd/system
+          fi
+
+          cp -r /usr/local/lib/scylla-operator/gke/systemd/. /host/usr/local/lib/systemd/system
+
+          if [[ ! -d /host/usr/local/lib/systemd/system/kube-node-configuration.service.d ]]; then
+            mkdir /host/usr/local/lib/systemd/system/kube-node-configuration.service.d
+          fi
+
+          cat<<-EOF > /host/usr/local/lib/systemd/system/kube-node-configuration.service.d/10-path.conf
+            [Service]
+            Environment="PATH=/var/lib/xfs-formatter/bin:$( chroot /host /bin/bash -c 'echo "${PATH}"' )"
+          EOF
+
+          if [[ ! -d /host/mnt/raid-disks ]]; then
+            mkdir /host/mnt/raid-disks
+          fi
+
+          if [[ ! -d /host/mnt/raid-disks/disk0 ]]; then
+            mkdir /host/mnt/raid-disks/disk0
+          fi
+          chmod a+w /host/mnt/raid-disks/disk0
+
+          chroot /host /bin/bash -euxo pipefail -c '
+              systemctl daemon-reload
+              systemctl enable --now mnt-raid\\x2ddisks-disk0.mount
+              systemctl start xfs-formatter.service
+              journalctl -xeu xfs-formatter -f
+              '
+        volumeMounts:
+        - name: hostfs
+          mountPath: /host
+          mountPropagation: Bidirectional
+        readinessProbe:
+          exec:
+            command:
+              - "chroot"
+              - "/host"
+              - "/bin/bash"
+              - "-euExo"
+              - "pipefail"
+              - "-c"
+              - "shopt -s inherit_errexit"
+              - "xfs_info \"$( grep -E '/mnt/raid-disks/disk0($| )' /proc/mounts | sed -E -e 's/([^ ]+) .+/\\1/' )\""
```

Comment:
nit: `- |` to avoid escaping quotes, ideally consistent command vs. args with https://github.com/scylladb/scylla-operator/pull/770/files#diff-c98bf62e26f5b0c21fccffc335ea032ac98cb51c11c757a32e6da6da19575bd4R31